data/syndicate jobs.txt

```diff
@@ -306,3 +306,183 @@ ship "Vanguard" "Vanguard Test Dummy"
 	add attributes
 		"automaton" 1
 		"self destruct" 1
+
+
+mission "Syndicate Prisoner Transport [0]"
+	name "Transport indicted executive to house arrest"
+	job
+	repeat
+	description `Transport an indicted white-collar criminal with his assistants, lawyers, and collection of <cargo> to <planet>, where he will await trial under "house arrest". Payment is <payment>.`
```

Comment:
Punctuation goes inside quotes in American English, no matter what.

Reply:
Fixed in 97f4eb0.